tasks/RedHat/generic/assert-systemd-tmpfiles.yml

```diff
@@ -3,18 +3,16 @@
 - debug:
     msg: "imported RedHat/generic/assert-systemd-tmpfiles.yml"
 
-- name: Check if file /etc/tmpfiles.d/sap.conf exists
-  shell: test -f /etc/tmpfiles.d/sap.conf
-  register: sap_conf_result
-  ignore_errors: yes
-  changed_when: no
+- name: Get info about file /etc/tmpfiles.d/sap.conf
+  stat:
+    path: /etc/tmpfiles.d/sap.conf
+  register: __stat_tmpfiles_sap_conf
 
 - name: Assert that file /etc/tmpfiles.d/sap.conf exists
   assert:
-    that:
-      - sap_conf_result.rc == 0
-    fail_msg: "FAIL: File '/etc/tmpfiles.d/sap.conf' does not exist!"
-    success_msg: "PASS: File '/etc/tmpfiles.d/sap.conf' exists."
+    that: __stat_tmpfiles_sap_conf.stat.exists == true
```

Comment:
don't compare to `true` - just evaulate the expression as a boolean
`that: __stat_tmpfiles_sap_conf.stat.exists | bool`

Reply:
Wouldn't `that: __stat_tmpfiles_sap_conf.stat.exists` (without the bool filter) also be fine? What could happen without the bool filter?

Reply:
> Wouldn't `that: __stat_tmpfiles_sap_conf.stat.exists` (without the bool filter) also be fine?

Yes, because `__stat_tmpfiles_sap_conf.stat.exists` is a "compound" variable, so not impacted by the ansible "bare variable" problem.

> What could happen without the bool filter?